src/Driver/Result.php

```diff
@@ -90,4 +90,9 @@ public function columnCount(): int;
      * Discards the non-fetched portion of the result, enabling the originating statement to be executed again.
      */
     public function free(): void;
+
+    /**
+     * Returns the column information.
+     */
+    public function getColumnMeta($index): array;
```

Comment:
This is a breaking change: We cannot add new methods to existing interfaces without breaking userland implementations.